tensorflow_datasets/core/file_format_adapter.py

```diff
@@ -386,6 +386,8 @@ def _item_to_tf_feature(item, key_name):
         "FeatureConnector should return a numpy array with the correct dtype "
         "instead of a Python list.".format(key_name)
     )
+  if isinstance(v, bytes) and not isinstance(v, str):
```

Comment:
Use `six.binary_type`. And this doesn't solve the issue if you pass a list of 0 strings (i.e. multiple all 0 strings)
(see my comment on #237; I think this is only an issue with all 0's).

Reply:
```python
if isinstance(v, six.bytes_type):
  ...
elif isinstance(v, (list, tuple)) and all((isinstance(x, six.bytes_type) for x in v)):
  ...
```
Would that be acceptable for dealing with multiple byte strings?